contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.3.0--2.4.0.sql

```diff
@@ -55,4 +55,20 @@ GRANT EXECUTE ON FUNCTION sys.degrees(TINYINT) TO PUBLIC;
 DROP PROCEDURE sys.babelfish_drop_deprecated_object(varchar, varchar, varchar);
 
 -- Reset search_path to not affect any subsequent scripts
-SELECT set_config('search_path', trim(leading 'sys, ' from current_setting('search_path')), false);
\ No newline at end of file
+SELECT set_config('search_path', trim(leading 'sys, ' from current_setting('search_path')), false);
```

Comment:
Create the view before calling set_config().

Reply:
Fixed in next revision by creating view before calling set_config().